src/services/content.tsx

```diff
@@ -1,3 +1,4 @@
+import API from "../utils/api";
```

Comment:
Please revert this file change as the PR title and task do not mention this change or else create a task and add in the title.

@deepali-chavhan